docs/integrations/sources/postgres.md

```diff
@@ -97,5 +97,82 @@ GRANT SELECT ON ALL TABLES IN SCHEMA <schema_name> TO airbyte;
 ALTER DEFAULT PRIVILEGES IN SCHEMA <schema_name> GRANT SELECT ON TABLES TO airbyte;
 ```
 
+#### 3. Set up CDC \(Optional\)
+
+Please read the section on CDC below for more information.
+
+#### 4. That's it!
+
 Your database user should now be ready for use with Airbyte.
 
+## Change Data Capture (CDC) / Logical Replication / WAL Replication
+We use [logical replication](https://www.postgresql.org/docs/10/logical-replication.html) of the Postgres write-ahead log (WAL) to incrementally capture deletes using the `pgoutput` plugin. 
+
+We do not require installing custom plugins like `wal2json` or `test_decoding`. We use `pgoutput`, which is included in Postgres 10+ by default.
+
+Please read the [CDC docs](../../architecture/cdc.md) for an overview of how Airbyte approaches CDC.
+
+### Should I use CDC for Postgres?
+* If you need a record of deletions and can accept the limitations posted below, you should to use CDC for Postgres.
+* If your data set is small and you just want snapshot of your table in the destination, consider using Full Refresh replication for your table instead of CDC.
+* If the limitations prevent you from using CDC and your goal is to maintain a snapshot of your table in the destination, consider using non-CDC incremental and occasionally reset the data and re-sync.
+
+### CDC Limitations
+* Make sure to read our [CDC docs](../../architecture/cdc.md) to see limitations that impact all databases using CDC replication.
+* CDC is only available for Postgres 10+.
+* Airbyte requires a replication slot configured only for its use. Only one source should be configured that uses this replication slot.
+* Log-based replication only works for master instances of Postgres.
+* Using logical replication increases disk space used on the database server. The additional data is stored until it is consumed.
```

Comment:
should we warn that if you don't set your replication frequency high enough or if you decide to stop replicating with airbyte, that you better delete the replication slot?

Reply:
as someone who isn't aware of all these and are reading this for the first time, that seems useful